# Simulation Params
## Robot parameters
simulation_speed: 1.0
target_robot_count: 10
obstacle_robot_count: 4
frame_rate: 30

## MAV parameters
vel_xy_p: 1.0
vel_xy_i: 0.0
vel_xy_d: 0.5
vel_xy_max: 2.0
vel_z_p: 2.0
vel_z_i: 0.5
vel_z_d: 1.0
vel_z_max: 0.25

## Robot Detection Params

# Debug Mode displays images and trackbars
debug_mode: true

# True uses webcam directly from opencv, false subscribes to camera/image_raw
use_webcam: false

# Morphological operations iterations for white filtering
pre_blur: 2
pre_erosions_w: 12
dilations_w: 25
post_erosions_w: 5


pre_erosions_g: 3
dilations_g: 20
post_erosions_g: 0


# HSV filter parameters for the white part of the robot
h_min_w: 0
h_max_w: 66
s_min_w: 0
s_max_w: 94
v_min_w: 188
v_max_w: 256

# HSV filter parameters for the green part of the robot
h_min_g: 39
h_max_g: 88
s_min_g: 54
s_max_g: 256
v_min_g: 30
v_max_g: 256

# HSV filter parameters for the red part of the robot
h_min_r: 0
h_max_r: 256
s_min_r: 0
s_max_r: 256
v_min_r: 0
v_max_r: 256

# Shape detection parameters
canny_thresh1: 1
canny_thresh2: 1
canny_aperture: 3
poly_area_min: 0
poly_area_max: 50000
max_dist: 50
